### PR TITLE
A post giving an example of how we interview

### PR DESCRIPTION
**Pull request by bobturneruk**

### CODE REVIEW

**Review by ns-rse — APPROVED** [review]
Nice and clear. :+1:

File: _posts/2022-05-23-interview.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+- What approaches could you use in this role to plan time on projects effectively and manage competing demands?
+- How would you advocate for the importance of Research Software Engineering as a role and software as a research output?
+- How would you motivate a reluctant academic to use version control?
+
+## After the interview
+
+After the interview, the panel will rank interviewees, again this is against the criteria in the person specification and the job description, before recommending that “Human Resources” make offers.
+
+<hr/>
```
Comment (ptheywood) [inline _posts/2022-05-23-interview.md]:
```suggestion
---
```

You can use 3 (or more) dahses, underscores or asterisks on their own line in markdown for horizontal rules.

**Review by mondus — APPROVED** [review]
I am happy with this but suggest we hold off publishing until we can release it for maximum impact.

**Comment by mondus** [comment]
ML post is now available so we should tailor this post to include the advert for our job: https://www.jobs.ac.uk/job/CRL803/research-software-engineer-rse-associate-rse-for-machine-learning

**Pull request merged**